src/config/swagger.py

```diff
@@ -0,0 +1,39 @@
+
+# Swagger template and configuration setup
+
+template = {
+    'swagger': '2.0',
+    'info': {
+        'title': 'Yakitabu backend',
+        'description': 'Yakitabu Book P2P Book Loan API',
+        'contact': {
+            'responsibleOrganization': '',
+            'responsibleDeveloper': '',
+            'email': 'yakitabu.io@yakitabu.com',
```

Comment:
Is this the correct email for this project? 😕

Reply:
No, its not the correct email. Its just some placeholder

Reply:
you can add the correct email then